### ISSUE TITLE
tencentcloud_clb_listener_rule health_check_switch always true

### ISSUE THREAD

**Issue by l10178** [terraform-providers/terraform-provider-tencentcloud#235]
### Terraform Version

0.12

### Affected Resource(s)

* tencentcloud_clb_listener_rule

### Terraform Configuration Files

```hcl
resource "tencentcloud_clb_listener_rule" "foo" {
  listener_id                = "xxx"
  clb_id                     = "xxx"
  domain                     = "foo.net"
  url                        = "/bar"
  health_check_switch        = false
  scheduler                  = "WRR"
}
```

 

### Expected Behavior

There health_check_switch should be FALSE.

### Actual Behavior

There health_check_switch is TRUE, and the others such as,  health_check_http_code is 31, health_check_http_method is HEAD.

### References

https://www.terraform.io/docs/providers/tencentcloud/r/clb_listener_rule.html

**Comment by gailwang** [comment]
Hi, the bug of the unchangeable health_check_switch will be fixed in the next version. While the  health_check_http_code and health_check_http_method are set as default value, those will not affect the health_check_switch itself. Thanks for your comment.

**Comment by gailwang** [comment]
Hello @I10178. This has been released in v1.25.0, please have a try, thanks.

**Comment by likexian** [comment]
Hello, this issue will be closed.
If you have any problems, please open a new issue.
